### PR TITLE
Timer

### PR DESCRIPTION
**Pull request by grantoesterling**
Adding Scheduler class that uses cron to manage time based messages :)

### CODE REVIEW

File: src/Classes/SchedulerBot.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import { Channel, Collection, EmbedBuilder, TextChannel } from 'discord.js'
+import { ProjectConfig } from '../ProjectConfig/projectConfig'
+import { artIndexerBot } from '..'
+import { delay } from './APIBots/utils'
+import { randomColor } from '../Utils/smartBotResponse'
+
+const cron = require('node-cron')
```
Comment (asaltste) [inline src/Classes/SchedulerBot.ts:7]: nit: should this be import cron from 'node-cron'

File: src/Classes/SchedulerBot.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+  async initialize() {
+    await delay(8000) // Wait for bot to connect and channels to load
+    console.log('Starting Scheduler')
+    const channels = this.channels
+    const projectConfig = this.projectConfig
+
+    // Birthdays
+    cron.schedule(
+      '0 1,9,17 * * *',
```
Comment (asaltste) [inline src/Classes/SchedulerBot.ts:29]: nit: add a comment about what this time is

File: src/Classes/SchedulerBot.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      },
+      null,
+      true,
+      'America/Chicago'
+    )
+
+    // Marfa
+    const sendMarfaMessage = this.sendMarfaMessage
+    cron.schedule(
+      '0 11 * * *',
```
Comment (asaltste) [inline src/Classes/SchedulerBot.ts:42]: nit: add a comment about what this time is

**Review by asaltste — APPROVED** [review]

**Review by mchrupcala — APPROVED** [review]

File: src/Classes/SchedulerBot.ts (hunk 1 above)
Comment (grantoesterling) [reply src/Classes/SchedulerBot.ts:7]: thats what i wanted to do, but for reasons I can't comprehend, the node-cron library only works this way :(

**Pull request merged**